### DISCUSSION TITLE
Limits on multi tenancy

### DISCUSSION THREAD

**Discussion by e-newt** [CC-in-the-Cloud/Admin#79] [General]
Hi all,
On Wednesday, I caught the tail end of discussion about the need to limit multi-tenancy. 

I want to share an example of how this is done for DOD SRG IL 5: 
![image](https://user-images.githubusercontent.com/39411588/218145683-c9c316e0-0296-47e5-b46e-6a171903cb68.png)

**Comment by jgb1128** [comment]
This relates to Issue #35 .  This is a deployment issue and most likely not a CC testing concern although the labs should ask the vendor for their deployment scenarios. Of note the pricing differential for BM vs MT is significant.
